tests/functional/error-bad-types/expected-stdout

```diff
@@ -1,5 +1,4 @@
 Creating ODBC connection to "Driver=SnowflakeDSIIDriver;Server=kebooladev.snowflakecomputing.com;Port=443;Tracing=0;Database="%s";Schema="%s"".
-Use schema "%s" in database "%s"
```

Comment:
Tento log som odstranil, kedze o riadok vyssie je connection string, kde je to vidiet + pri kazdom retry sa vytvara aj nove spojenie (ak spadlo) a bolo to otravne, ze to stale vypisovalo tuto hlasku (pri kazdom retry).